site/topics/objects/introduction.rst

```diff
@@ -82,255 +82,207 @@ Starting the Class
-   
-    .. code-block:: python
-        :linenos:
-	  
-        #from EnthusiasticStudent import *    # Only need this if using multiple files in IDE
-        a_student = EnthusiasticStudent()
-   
-    1. Figure out how to get the first name from ``a_student``
+Constructor/Instantiation and Attributes
+----------------------------------------
+
+* With just the ``class Circle``, we can actually start making instances of the class
 
-    2. What is the ``type`` of ``a_student``? The ``type`` of ``EnthusiasticStudent``? How about ``EnthusiasticStudent()``?
+.. code-block:: python
+    :linenos:
 
-    3. Create a new, different student, but this time set the first name to something else. Check its attributes. 
+    some_circle = Circle()
+    print(type(some_circle))    # Results in something like <class '__main__.Circle'>
 
-    4. Add a print statement saying "im running from inside a constructor" to your constructor right below where we assigned the attributes. Re-run the code above. What happened? Why?
 
-* We can put whatever code we want in the constructor. Just in our case, setting the attributes is enough to get what we want done.
-* **NOTE** ``self`` is pretty important here. If you forget the self, would you be creating an attribute for the object, or a local variable for the constructor?   
+* However, this is not particularly useful as we have yet to really describe the ``Circle`` class
+* To start making the class useful, we will write a special function that tells Python how to setup the class for our needs
+
+.. code-block:: python
+    :linenos:
 
+    class Circle:
+        """
+        A class for representing circle based on its radius. The class provides functionality to perform basic geometry
+        calculations (diameter, area, circumference).
+        """
 
-3. Let's add some functions/methods to this class.
+        def __init__(self):
+            """
+            Creates a Circle object with a radius of 0.
+            """
+            self.radius = 0
 
-    .. code-block:: python
-        :linenos:
-      
-        class EnthusiasticStudent:
-            '''
-            Obv we'll include a nice comment at the top of the class to explain what it's for... right?!
 
-            This EnthusiasticStudent is being used to demonstrate how we can create our own Objects.
-		  
-            It's going to have a few attributes and some simple functions.
-            '''
+* In the above example, we see the use of the special function called ``__init__``, which describes how to initialize an instance of the class
 
-            def __init__(self, first_name='John', last_name='Doe', student_num='000000000', current_avg=0):
-                self.first_name = first_name 
-                self.last_name = last_name
-                self.student_num = student_num
-                self.current_avg = current_avg
+    * The ``__init__`` method is called the *instantiation* method, but is commonly referred to as the *constructor*
```

Comment:
`__init__` is the initializer/constructor. `__new__` actually instantiates the class.